services/QuillLMS/lib/tasks/migrate_subscriptions.rake

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+namespace :migrate_subscriptions do
+  desc 'migrates all subscriptions of one type to another type'
+  task :run, [:old_sub, :new_sub] => :environment do |t, args|
```

Comment:
I'm a little torn on this but I wonder if it might be good to just have the `old_sub` `new_sub` thing as a hash in this file rather than args? That makes it less reusable in the future but gives us clearer documentation in the code of what we changed. Totally up to you, just wanted to flag it.

Reply:
Yeah after thinking about it, I think you're right that it's more readable and better for documentation if we put it as a hash. Seeing as this isn't an action we'll be performing very often if ever at all, I think it's okay to sacrifice code flexibility to make it more readable!